docs/9.0/guides/getting-started/hello-world.md

```diff
@@ -19,6 +19,10 @@ Rather than display "Hello, World!", the demo contains 100 rows and 13 columns p
 
 Want to play with the code yourself? Select "Open Sandbox" in the frame's bottom right corner.
 
-The demos for Angular and Vue are coming in June 2021, stay tuned!
+The demos for Angular and Vue are coming soon, stay tuned!
 
-<HelloWorld></HelloWorld>
+<HelloWorld :sandboxesIds="[
```

Comment:
I might be wrong, but as this array is not going to change dynamically, Vue does not have to watch it for changes. We may skip dynamic bind. But needs testing
```suggestion
<HelloWorld sandboxesIds="[
```

Reply:
It would consider attribute value as a string. I would leave it as it is. WDYT, @wojciechczerniak ?

Reply:
🤔 I thought it would be the same but not proxied. Ok then